src/autoscaler/eventgenerator/generator/evaluator_test.go

```diff
@@ -613,6 +607,7 @@ var _ = Describe("Evaluator", func() {
 							errorStr = errorStr + tmp
 						}
 						scalingEngine.RouteToHandler("POST", urlPath, ghttp.RespondWithJSONEncoded(http.StatusBadRequest, errorStr))
+						Expect(triggerChan).To(BeSent(triggerArrayGT))
```

Comment:
@qibobo I tried to find a reason why it's pended however I haven't found it.
I was wondering if the below story is the first commit to add this file. Do you know/remember the reason?
https://www.pivotaltracker.com/n/projects/1566795/stories/119505309